scripts/rosversion

```diff
@@ -68,47 +67,60 @@ def get_distro_name_from_roscore():
         traceback.print_exc()
 
 
-if len(sys.argv) == 2:
-    stack_name = sys.argv[1]
+def print_without_newline(argtext):
+    """Print with no new line."""
+    print(argtext, end='')
 
-    if stack_name == '-d' or stack_name == '--distro':
-        if 'ROS_DISTRO' in os.environ:
-            distro_name = os.environ['ROS_DISTRO']
-        else:
-            distro_name = get_distro_name_from_roscore()
-        if not distro_name:
-            print('<unknown>')
-        else:
-            print(distro_name)
-        sys.exit(0)
 
-    rosstack = rospkg.RosStack()
-    try:
-        version = rosstack.get_stack_version(stack_name)
-    except rospkg.ResourceNotFound as e:
-        try:
-            # hack to make it work with wet packages
-            mm = ManifestManager(PACKAGE_FILE)
-            path = mm.get_path(stack_name)
-            package_manifest = os.path.join(path, 'package.xml')
-            if os.path.exists(package_manifest):
-                from xml.etree.ElementTree import ElementTree
-                try:
-                    root = ElementTree(None, package_manifest)
-                    version = root.findtext('version')
-                except Exception:
-                    pass
-        except rospkg.ResourceNotFound as e:
-            print("Cannot locate [%s]" % (stack_name))
-            sys.exit(1)
+parser = argparse.ArgumentParser(
+    description='rosversion -d: Output the version of the given package\n'
+    'rosversion package: ROS distribution name',
```

Comment:
These two messages don't look consistent to me. The first one clearly says "Output the version of ...". The second one just says "ROS distribution name".

Please make the messages similar, e.g.:

> rosversion -d: Output the version of the given package
> rosversion package: Output the ROS distribution name

Reply:
See 8dac7a11503f7f598fcb36125eff0f0f8d2f13d7.